src/lib/rule-finder.js

```diff
@@ -30,32 +30,116 @@ function _getCurrentNamesRules(config) {
   return Object.keys(config.rules);
 }
 
-function _normalizePluginName(name) {
-  const scopedRegex = /(@[^/]+)(\/(.+))?/;
-  const match = scopedRegex.exec(name);
+let eslintNaming;
+// ignore function from coverage because it varies per eslint version
```

Comment:
If I need to update this for proper code coverage please advise how to do so.

Reply:
We could move it to a separate file, and istanbul ignore that file?

Reply:
I actually just did this and was about to push.

Reply:
Done!